code/modules/mob/living/brain/brain_item.dm

```diff
@@ -323,6 +323,7 @@
 	name = "alien brain"
 	desc = "We barely understand the brains of terrestial animals. Who knows what we may find in the brain of such an advanced species?"
 	icon_state = "brain-x"
+	organ_traits = list(TRAIT_CAN_STRIP, TRAIT_LITERATE)
```

Comment:
```suggestion
	organ_traits = list(TRAIT_CAN_STRIP)
```

Xenomorphs should not be able to read or write.

Reply:
from science fiction, though I couldn't find a definitive answer, xenomorphs seems to be pretty smart.

Reply:
I played too much Colonial Marines back in the day, but I feel like this breaks the traditional xenomorph lore and flavor.

When I made the PR for literacy, I specifically was aiming to make it granted _exclusively_ to human subtypes. (with a few rare exceptions)  I really don't feel like xenomorphs should be included into that category, but if you are deadset on this, then at least limit it to only xeno queens.